src/plugins/data/common/search/expressions/exists_filter.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ExecutionContext } from 'src/plugins/expressions/common';
+import { functionWrapper } from './utils';
+import { existsFilterFunction } from './exists_filter';
+
+describe('interpreter/functions#existsFilter', () => {
+  const fn = functionWrapper(existsFilterFunction);
+  let context: ExecutionContext;
+
+  beforeEach(() => {
+    context = {
+      getSearchContext: () => ({}),
+      getSearchSessionId: () => undefined,
+      types: {},
+      variables: {},
+      abortSignal: {} as any,
+      inspectorAdapters: {} as any,
+    };
+  });
+
+  it('returns an object with the correct structure', () => {
+    const actual = fn(null, { field: { spec: { name: 'test' } } }, context);
```

Comment:
I understand this is copy-paste from similar tests in this plug, but I've seen it a lot and wanted to suggest a small improvement. It is better to not create global module variables, like `context` here and not use `beforeEach` utility, which is specific to Jest. This can be done like so:

```suggestion
    const context = setup();
    const actual = fn(null, { field: { spec: { name: 'test' } } }, context);
```

Simply `setup()` function that creates everything needed for each test case every time.